grammars/README.md

```diff
@@ -1,11 +1,4 @@
 ## Updating the grammar
 
 When making changes to `dart.json`, please either make PRs or file issues for
-the following known repos that contain a copy of this file:
-
-* https://github.com/Dart-Code/Dart-Code/blob/master/syntaxes/dart.json
-
-## Note
-
-File dart.json was adapted from:
-https://github.com/dart-atom/dartlang/blob/master/grammars/dart.cson
+the projects listed in the main [readme.md](../readme.md).
```

Comment:
Some of the projects below don't need notifications / issues created; I believe they just periodically take the latest from head for their supported languages.

I'm not sure if we need to note that here; creating issues across 4 repos is expensive enough that most people may not do it.

Can we version the grammar? Perhaps as a field or comment in the `dart.json` file? That way people that consume copies of the file will know if they're up to date or not.

Reply:
> Some of the projects below don't need notifications / issues created; I believe they just periodically take the latest from head for their supported languages.

Oh yeah, good point. I was trying to avoid duplicating but maybe it doesn't make sense. DevTools is definitely one place not automatically updating though - I can revert this and just list Dart-Code + DevTools here.

> Can we version the grammar? Perhaps as a field or comment in the `dart.json` file?

Yeah, that makes sense - I'll add that too. I don't know that we can easily enforce it being increased (or changelog being bumped). Perhaps it could be useful to publish this package (also with a version) and instead of manually copying, projects like DevTools could reference it? I don't know if that would add more overhead than necessary though.

Reply:
I've reverted the changes to the nested readme (but added DevTools), and added a version. I can't find anything in textmate spec about versions so I'm assuming putting an "x.y.z" field called "version" here is valid (I also confirmed VS Code just ignores it and continues to use the grammar).

I also added a changelog since if we're having numbers it may be useful to see a summary of what changed/dates.